cob_bringup/robots/base_solo.launch

```diff
@@ -8,26 +8,26 @@
 	<include file="$(arg pkg_hardware_config)/common/upload_robot.launch" >
 		<arg name="robot" value="$(arg robot)" />
 	</include>
+    <include file="$(find cob_default_robot_config)/upload_param.launch" >
+		<arg name="robot" value="$(arg robot)" />
+	</include>
```

Comment:
because you need the parameter  **/base/service_ns** to initialise the base using teleop